src/index.js

```diff
@@ -1,14 +1,15 @@
+/* global document */
```

Comment:
Without it eslint returns error that global is undefined variable - [no-undef](http://eslint.org/docs/rules/no-undef)